test/foundry/new/helpers/FuzzEngineLib.sol

```diff
@@ -233,38 +240,44 @@ library FuzzEngineLib {
 
     function getNativeTokensToSupply(
         FuzzTestContext memory context
-    ) internal pure returns (uint256) {
+    ) internal view returns (uint256) {
         uint256 value = 0;
 
         for (uint256 i = 0; i < context.orders.length; ++i) {
             OrderParameters memory orderParams = context.orders[i].parameters;
             for (uint256 j = 0; j < orderParams.offer.length; ++j) {
                 OfferItem memory item = orderParams.offer[j];
 
-                // TODO: support ascending / descending
-                if (item.startAmount != item.endAmount) {
-                    revert(
-                        "FuzzEngineLib: ascending/descending not yet supported"
-                    );
-                }
-
                 if (item.itemType == ItemType.NATIVE) {
-                    value += item.startAmount;
+                    if (item.startAmount != item.endAmount) {
+                        value += _locateCurrentAmount(
+                            item.startAmount,
+                            item.endAmount,
+                            orderParams.startTime,
+                            orderParams.endTime,
+                            true
```

Comment:
I think this should be false for offer items?

Reply:
Ah, whoops, looks like they're reversed.